tests/e2e/reborn_webui_harness.py

```
@@ -453,7 +466,24 @@ async def send_and_settle(
+    submit_body: dict = {}
+    for _ in range(12):
+        submit_body = await _submit_message(client, base_url, thread_id, content)
+        outcome = submit_body.get("outcome")
+        if outcome in ACCEPTED_SEND_OUTCOMES:
+            break
+        if outcome == "rejected_busy":
+            await asyncio.sleep(0.5)
+            continue
+        raise AssertionError(
+            f"Message was not accepted for a run; outcome={outcome!r}, body={submit_body}"
+        )
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)